qa/rpc-tests/bumpfee.py

```diff
@@ -213,6 +213,10 @@ def test_unconfirmed_not_spendable(rbf_node, rbf_node_address):
     assert bumpid not in rbf_node.getrawmempool()
     assert rbfid in rbf_node.getrawmempool()
 
+    # call abandontransaction to make sure the wallet does not add the bump
+    # transaction back into the mempool.
+    rbf_node.abandontransaction(bumpid)
```

Comment:
Oops, yes, I got confused about what the test was doing with the two transactions. Fixed and updated the comment in a1046e099e4fac8b82d6e268eeb9fb97e4a9c9ef.